### PR TITLE
Java is still being updated, trying to call out all the java package prefixes

### PR DESCRIPTION
**Pull request by agirlnamedsophia**
/domain @Betterment/squad-sre @ansongomes 
/platform @samandmoore 

Spun up an instance with the new AMI and java is still updated to a security patch.

### CODE REVIEW

File: tasks/level-1/1.8.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   yum:
     name: "*"
     state: latest
-    exclude: nginx*,java*
+    exclude: nginx*,java*,jre*,jdk*
```
Comment (devinburnette) [inline tasks/level-1/1.8.yml:10]: maybe `*jre*`, and `*jdk*`

Comment (ansongomes) [reply tasks/level-1/1.8.yml:10]: Actually unless I am missing something, I am more for not including `jre` to this exclusion list. The Runtime is where all the vulnerabilities are exploited in Java and I'd really like to stay up-to-date on this. 

Also, do we really need to install and run the JRE if we already have JDK on the system? Technically, our JDK should be the only executable running Java on the system and not the JVM. Please let me know if I am wrong about this or if I am missing something here.

**Comment by agirlnamedsophia** [comment]
nanda?

**Comment by Cbeck527** [comment]
<< domain lgtm

:fried_egg:

**Comment by samandmoore** [comment]
<<platformlgtm

**Pull request merged**